docs/sources/api/_index.md

```diff
@@ -621,6 +621,17 @@ backing store. Mainly used for local testing.
 
 In microservices mode, the `/flush` endpoint is exposed by the ingester.
 
+### Tell ingester to release all resources on next SIGTERM
+
+```
+POST /ingester/prepare_shutdown
+```
+
+`/ingester/prepare_shutdown` will prepare the ingester to release resources on the next SIGTERM,
```

Comment:
I feel like we should define SIGTERM here.

Reply:
SIGTERM is a type of signal that a process can receive and is supported by all Unix-based operating systems. Do you think saying `SIGTERM signal` is enough here?

Reply:
LOL, and I have just exposed my UNIX ignorance. But it's also possible that we might have customers in Windows-only environments who could use the information?  What do you think?

Reply:
Sorry, with "could use the information" do you mean the definition of SIGTERM? If so: personally, I feel like this isn't the appropriate place for explaining a concept like SIGTERM, although saying "SIGTERM signal" indeed looked better than just saying "SIGTERM".

Reply:
OK, let's go with "SIGTERM signal" then.